packages/testing/src/test/Helpers.test.ts

```diff
@@ -0,0 +1,103 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { HierarchyCacheMode, initialize, PresentationTestingInitProps, terminate } from "../presentation-testing/Helpers";
+import * as sinon from "sinon";
+import { Presentation as PresentationBackend } from "@itwin/presentation-backend";
+import { Presentation as PresentationFrontend } from "@itwin/presentation-frontend";
+import * as rimraf from "rimraf";
+import { Guid } from "@itwin/core-bentley";
+import { NoRenderApp } from "@itwin/core-frontend";
+
+describe("Helpers", () => {
+  describe("initialize", () => {
+    afterEach(async () => {
+      await terminate();
+    });
+
+    it("initializes PresentationBackend and PresentationFrontend on initialize", async () => {
+      const backendInitializationSpy = sinon.spy(PresentationBackend, "initialize");
+      const frontendInitializationSpy = sinon.spy(PresentationFrontend, "initialize");
```

Comment:
I suggest we stub **all** these `initialize`, `startup` and `terminate` calls in `beforeEach` - that we the real ones won't be called, thus speeding up the tests.